Tests/UnitTests/Core/Seeding/BinnedGroupTest.cpp

```diff
@@ -0,0 +1,256 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/unit_test.hpp>
+
+#include "Acts/Seeding/BinnedGroup.hpp"
+#include "Acts/Utilities/Grid.hpp"
+#include "Acts/Utilities/GridBinFinder.hpp"
+#include "Acts/Utilities/GridIterator.hpp"
+
+#include <array>
+#include <memory>
+#include <vector>
+
+namespace Acts::Test {
+
+BOOST_AUTO_TEST_CASE(binned_group_constructor) {
```

Comment:
This test is not actually testing anything, is it?

Reply:
true ... a remnant of previous versions of the contructors

Reply:
well... not really. This is checking all the constructors are not having problems (including copy and handling the navigation)

Reply:
I guess it's a compile test.